src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -136,7 +152,47 @@ def get(self, svc_id):
         daemon['rgw_metadata'] = metadata
         daemon['rgw_status'] = status
         return daemon
-
+    
+    @RESTController.Collection(method='GET', path='/multisite')
+    @allow_empty_body
+    def get_multisite_info(self):
+        multisite_info = []
+        try:
+            instance = RgwClient.admin_instance()
+        except NoRgwDaemonsException as e:
+            raise DashboardException(e,http_status_code=404, component='rgw')
+        multisite_info = instance.get_multiste_info()
+        return multisite_info
+    
+    @RESTController.Collection(method='POST', path='/realm', status=201)
+    @allow_empty_body
+    def create_realm(self, realm_name, default):
+        try:
+            instance = RgwClient.admin_instance()
+            result = instance.create_realm(realm_name, default)
+            return result
+        except NoRgwDaemonsException as e:
+            raise DashboardException(e, http_status_code=500, component='rgw')
```

Comment:
how come the status code is 500? Shouldn't it be 404? 
As per this [guide](https://restfulapi.net/http-status-codes/), if a resource is not found then its a 404.